applications/DEMApplication/custom_strategies/strategies/explicit_solver_continuum.cpp

```diff
@@ -249,7 +249,7 @@ namespace Kratos {
 
         }
         //Synch this var.
-        r_model_part.GetCommunicator().MaxAll(r_process_info[SEARCH_CONTROL]);
+        r_process_info[SEARCH_CONTROL] = r_model_part.GetCommunicator().GetDataCommunicator().MaxAll(r_process_info[SEARCH_CONTROL]);
```

Comment:
this changed in behaviour? :-(

the new behaviour is nicer, however we'll need to be extra careful on this

Reply:
Why did this change? Just curious

Reply:
The point is that the DataCommunicator "grew" from the old python mpi interface, which had this behavior. I decided to keep it, since it is at least consistent between python and C, but it does clash with the Commmunicator behavior here...